cfgov/housing_counselor/tests/management/test_enforcement_list_saver.py

```diff
@@ -0,0 +1,18 @@
+import os
+from unittest import TestCase
+
+from housing_counselor.enforcement_list_saver import save_list
+
+
+class TestEnforcementListSaver(TestCase):
+    file_path = './zip_destination.zip'
+
+    def tearDown(self):
```

Comment:
A potentially cleaner alternative to this would be to use [`tempfile.TemporaryDirectory`](https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory), like:

```py
with TemporaryDirectory() as tempdir:
    zipfile = os.path.join(tempdir, 'archive.zip')
    save_list(content, zipfile)
    self.assertTrue(os.path.exists(zipfile))
```

This takes care of cleanup and guarantees that the file won't exist at the start of the test, avoiding the need for the `assertFalse` call.

Reply:
`TemporaryDirectory` isn't available in [Python 2.7](https://docs.python.org/2.7/library/tempfile.html#tempfile.TemporaryDirectory), unfortunately.  I've pushed an update that uses a lower-level method in the same library. Does that look better?

Reply:
Ah, that is too bad. Unfortunately, in that case, the `os.remove` calls probably should be in the test `tearDown`, as you had originally. Otherwise, if the assert statement fails, the test will abort, and those commands won't get run. If they're in `tearDown`, they'll happen no matter what.

Another option that's compatible with Python 2.7 is to use `addCleanup`:

```py
class Tests(TestCase):
    def setUp(self):
        self.tempdir = tempfile.mkdtemp()
        self.addCleanup(shutil.rmtree, self.tempdir)

    def testSomething(self):
        zipfile = os.path.join(self.tempdir, 'archive.zip')
        ...
```

Reply:
Thanks for the suggestion. Updated to include this.